helpdesk_ticket_close_inactive/readme/CONFIGURE.md

```diff
@@ -0,0 +1,11 @@
+
+To configure this module, you need to:
+
+- Go to Helpdesk > Settings > Teams.
+- Select a team.
+- Enable 'Auto Close Inactive Ticket' option.
```

Comment:
Enable 'Automatic closure of inactive tickets' option.

Reply:
Fixed. Thanks!